### PR TITLE
do not show solidity

### PR DESCRIPTION
**Pull request by bunsenstraat**
only load solidity with code if not another plugin is activated

### CODE REVIEW

File: apps/remix-ide/src/app.js [hunk 1]
```diff
@@ -362,7 +362,7 @@ class AppComponent {
             } catch (e) {
               console.log(e)
             }
-            if (params.code) {
+            if (params.code && (!params.activate || params.activate==='solidity')) {
```
Comment (yann300) [inline apps/remix-ide/src/app.js:365]: i think the issue is also that we need to show the activated plugin.
also I think `activate` is a comma separated list of plugins

Comment (bunsenstraat) [reply apps/remix-ide/src/app.js]: @yann300 should be fine now

Comment (yann300) [reply apps/remix-ide/src/app.js]: can you checl that if you load https://remix.ethereum.org/?#activate=starkNet_compiler&code=IyBEZWNsYXJlIHRoaXMgZmlsZSBhcyBhIFN0YXJrTmV0IGNvbnRyYWN0LgolbGFuZyBzdGFya25ldAoKZnJvbSBzdGFya3dhcmUuY2Fpcm8uY29tbW9uLmNhaXJvX2J1aWx0aW5zIGltcG9ydCBIYXNoQnVpbHRpbgoKIyBEZWZpbmUgYSBzdG9yYWdlIHZhcmlhYmxlLgpAc3RvcmFnZV92YXIKZnVuYyBiYWxhbmNlKCkgLT4gKHJlcyA6IGZlbHQpOgplbmQKCiMgSW5jcmVhc2VzIHRoZSBiYWxhbmNlIGJ5IHRoZSBnaXZlbiBhbW91bnQuCkBleHRlcm5hbApmdW5jIGluY3JlYXNlX2JhbGFuY2V7CiAgICAgICAgc3lzY2FsbF9wdHIgOiBmZWx0KiwgcGVkZXJzZW5fcHRyIDogSGFzaEJ1aWx0aW4qLAogICAgICAgIHJhbmdlX2NoZWNrX3B0cn0oYW1vdW50IDogZmVsdCk6CiAgICBsZXQgKHJlcykgPSBiYWxhbmNlLnJlYWQoKQogICAgYmFsYW5jZS53cml0ZShyZXMgKyBhbW91bnQpCiAgICByZXR1cm4gKCkKZW5kCgojIFJldHVybnMgdGhlIGN1cnJlbnQgYmFsYW5jZS4KQHZpZXcKZnVuYyBnZXRfYmFsYW5jZXsKICAgICAgICBzeXNjYWxsX3B0ciA6IGZlbHQqLCBwZWRlcnNlbl9wdHIgOiBIYXNoQnVpbHRpbiosCiAgICAgICAgcmFuZ2VfY2hlY2tfcHRyfSgpIC0+IChyZXMgOiBmZWx0KToKICAgIGxldCAocmVzKSA9IGJhbGFuY2UucmVhZCgpCiAgICByZXR1cm4gKHJlcykKZW5k&optimize=false&runs=200&evmVersion=null&version=soljson-v0.8.7+commit.e28d00a7.js

is the startnet plugin activated and displayed?

Comment (bunsenstraat) [reply apps/remix-ide/src/app.js]: it only activates it. but this is live, my fix is not deployed yet...

**Comment by bunsenstraat** [comment]
@yann300 https://ipfs.remixproject.org/ipfs/QmXhjod5aC2nqozbWakdbipe11Z2cJHkVCYSycrtgmqdcM/?#activate=starkNet_compiler&code=IyBEZWNsYXJlIHRoaXMgZmlsZSBhcyBhIFN0YXJrTmV0IGNvbnRyYWN0LgolbGFuZyBzdGFya25ldAoKZnJvbSBzdGFya3dhcmUuY2Fpcm8uY29tbW9uLmNhaXJvX2J1aWx0aW5zIGltcG9ydCBIYXNoQnVpbHRpbgoKIyBEZWZpbmUgYSBzdG9yYWdlIHZhcmlhYmxlLgpAc3RvcmFnZV92YXIKZnVuYyBiYWxhbmNlKCkgLT4gKHJlcyA6IGZlbHQpOgplbmQKCiMgSW5jcmVhc2VzIHRoZSBiYWxhbmNlIGJ5IHRoZSBnaXZlbiBhbW91bnQuCkBleHRlcm5hbApmdW5jIGluY3JlYXNlX2JhbGFuY2V7CiAgICAgICAgc3lzY2FsbF9wdHIgOiBmZWx0KiwgcGVkZXJzZW5fcHRyIDogSGFzaEJ1aWx0aW4qLAogICAgICAgIHJhbmdlX2NoZWNrX3B0cn0oYW1vdW50IDogZmVsdCk6CiAgICBsZXQgKHJlcykgPSBiYWxhbmNlLnJlYWQoKQogICAgYmFsYW5jZS53cml0ZShyZXMgKyBhbW91bnQpCiAgICByZXR1cm4gKCkKZW5kCgojIFJldHVybnMgdGhlIGN1cnJlbnQgYmFsYW5jZS4KQHZpZXcKZnVuYyBnZXRfYmFsYW5jZXsKICAgICAgICBzeXNjYWxsX3B0ciA6IGZlbHQqLCBwZWRlcnNlbl9wdHIgOiBIYXNoQnVpbHRpbiosCiAgICAgICAgcmFuZ2VfY2hlY2tfcHRyfSgpIC0+IChyZXMgOiBmZWx0KToKICAgIGxldCAocmVzKSA9IGJhbGFuY2UucmVhZCgpCiAgICByZXR1cm4gKHJlcykKZW5k&optimize=false&runs=200&evmVersion=null&version=soljson-v0.8.7+commit.e28d00a7.js

**Review by yann300 — APPROVED** [review]

**Pull request merged**